tests/less/comments/in-value.less

```diff
@@ -0,0 +1,14 @@
+.real-world-example {
+  background: radial-gradient(
+      circle at left 0% bottom $position,
+      transparent,
+      transparent $diameter,
+      #fbfbfb calc(#{$diameter} + 1px) // Add 1px for edge-smoothing.
+    );
+}
+
+.simplification { 
+  foo: (
+    calc() // not a comment anymore
+  );
+}
```

Comment:
It's been a while.. but it's not suppose to check for new lines here?

Reply:
Some tests file can't add it